packages/e2e-test-utils-playwright/src/request/comments.js

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
This file modeled after https://github.com/WordPress/gutenberg/blob/3dddb153c955ecb69336298e3384195c1ad47cdd/packages/e2e-test-utils-playwright/src/request/posts.js.